sdk/ai/azure-ai-projects/samples/agents/async_samples/sample_agents_vector_store_batch_enterprise_file_search_async.py

```diff
@@ -41,15 +45,24 @@ async def main():
 
         # We will upload the local file to Azure and will use it for vector store creation.
         _, asset_uri = project_client.upload_file("../product_info_1.md")
-        ds = VectorStoreDataSource(asset_identifier=asset_uri, asset_type=VectorStoreDataSourceAssetType.URI_ASSET)
-        vector_store = await project_client.agents.create_vector_store_and_poll(file_ids=[], name="sample_vector_store")
+        ds = VectorStoreDataSource(
+            asset_identifier=asset_uri,
+            asset_type=VectorStoreDataSourceAssetType.URI_ASSET,
+        )
+        vector_store = await project_client.agents.create_vector_store_and_poll(
+            file_ids=[], name="sample_vector_store"
+        )
         print(f"Created vector store, vector store ID: {vector_store.id}")
 
         # add the file to the vector store or you can supply file ids in the vector store creation
-        vector_store_file_batch = await project_client.agents.create_vector_store_file_batch_and_poll(
-            vector_store_id=vector_store.id, data_sources=[ds]
+        vector_store_file_batch = (
+            await project_client.agents.create_vector_store_file_batch_and_poll(
```

Comment:
Since I would like to tag your code in README, let's reduce the number of code here...
I think you can take out the round bracket.
But I also wonder could you pass data_source to create_vector_store_and_poll instead?

Reply:
Looks liked I can tag in another sample.   I don't need to tag here.   It s ok to call create_vector_store_file_batch_and_poll.   But please remove ()

Reply:
The problem is that the () are added by the linter "black", which we run to check our code, and it is unhappy because the line is too long. Any changes with continuation of line are being reformatted to string in brackets.